clients/src/main/java/org/apache/kafka/clients/consumer/internals/AsyncKafkaConsumer.java

```diff
@@ -319,7 +314,7 @@ private StreamsRebalanceData streamsRebalanceData() {
         }
 
         private StreamsRebalanceListener streamsGroupRebalanceCallbacks() {
```

Comment:
Should `streamsGroupRebalanceCallbacks` be reamed to `StreamsRebalanceListener`?

Reply:
@chia7712 the first letter may need to be lower case: `streamsRebalanceListener`.

Reply:
> the first letter may need to be lower case:

yes - please ignore my typo :)

Reply:
Thanks for finding this!
Fixed it!